attestation-service/verifier/src/csv/mod.rs

```diff
@@ -40,58 +39,60 @@ pub struct CsvVerifier {}
 impl Verifier for CsvVerifier {
     async fn evaluate(
         &self,
-        nonce: String,
-        attestation: &Attestation,
+        evidence: &[u8],
+        report_data: Option<&[u8]>,
+        init_data_hash: Option<&[u8]>,
```

Comment:
Hi @fangbaoshun  Hi Baoshun, I've changed the API of verifier of `CSV`. PTL if it looks good to you.

BTW, it will now change the behavior of current CoCoAS.

Reply:
LGTM.